.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "Bug report \U0001F41B"
+about: "Create a bug report if something isn't working as expected \U0001F61E"
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Thanks for using the Firebase App Distribution Fastlane plugin! Find a bug? Please fill in the template below and we'll look into it. -->
+
+### Issue Description
+<!-- Please include what's happening, expected behavior, and any relevant code samples -->
+
+##### Configuration
+<!--  What parameters are you passing to the `firebase_app_distribution` action? -->
+
+##### Complete verbose output
+<!-- 
+To print verbose output, set the `debug` parameter of the  `firebase_app_distribution` action to` true`.
+Caution: The output with `debug` set to true could contain sensitive data such as email addrresses, app ids, and/or project numbers. Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue 
+-->
+
+##### Environment
+<!-- What version of `fastlane` are you using? What version of `fastlane-plugin-firebase_app_distribution` are you using?  Note your issue may already be fixed in the latest version. The latest version can be found at https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/releases -->
+
+| Syntax      | Description |
```

Comment:
Should these headers be set to "Library" and "Version" or something?

Reply:
Ahh yeah my bad a little leftover from copy pasta 😞